model/project_parser_db.go

```diff
@@ -55,13 +55,21 @@ func ParserProjectFindOneById(id string) (*ParserProject, error) {
 }
 
 func ParserProjectFindOneByVersion(projectId string, version string) (*ParserProject, error) {
+	msg := message.Fields{
+		"message":        "retrieving parser project by version",
+		"project_id":     projectId,
+		"lookup_version": false,
+	}
 	if version == "" {
 		lastGoodVersion, err := FindVersionByLastKnownGoodConfig(projectId, -1)
 		if err != nil || lastGoodVersion == nil {
 			return nil, errors.Wrapf(err, "Unable to retrieve last good version for project '%s'", projectId)
 		}
 		version = lastGoodVersion.Id
+		msg["lookup_version"] = true
 	}
+	msg["version"] = version
+	grip.Info(msg)
```

Comment:
Let's only log for error (and log under grip.Debug) or else we'll be logging like crazy.